src/5e-SRD-Traits.json

```diff
@@ -524,9 +524,6 @@
     ],
     "subraces": [],
     "name": "Draconic Ancestry (Black)",
-    "desc": [
```

Comment:
Given that all other traits have descriptions, it would probably make more sense to make this an empty array rather than removing it completely, so the schema is consistent.

Reply:
I have no issues with this. I'll implement the change tomorrow if I have the time.